examples/inspection/plot_linear_model_coefficient_interpretation.py

```diff
@@ -305,28 +310,32 @@
 from sklearn.model_selection import cross_validate
 from sklearn.model_selection import RepeatedKFold
 
+cv = RepeatedKFold(n_splits=5, n_repeats=5, random_state=0)
 cv_model = cross_validate(
     model,
     X,
     y,
-    cv=RepeatedKFold(n_splits=5, n_repeats=5),
+    cv=cv,
     return_estimator=True,
     n_jobs=2,
 )
+
 coefs = pd.DataFrame(
     [
-        est.named_steps["transformedtargetregressor"].regressor_.coef_
-        * X_train_preprocessed.std(axis=0)
-        for est in cv_model["estimator"]
+        est[-1].regressor_.coef_ * est[:-1].transform(X.iloc[train_idx]).std(axis=0)
```

Comment:
Here, it should be better to normalize considering the training set of each fold instead of the external training set. There is some data leakage otherwise.